packages/apptive_grid_core/lib/src/network/apptive_grid_client.dart

```diff
@@ -408,7 +408,7 @@ class ApptiveGridClient extends ChangeNotifier {
     ApptiveGridFilter? filter,
     bool isRetry = false,
     Map<String, String> headers = const {},
-    int? pageSize,
+    PagingRequest? pagingRequest,
```

Comment:
I think it would be fine to just have separate parameters for `page` and `pageSize` here to limit overhead.
Also out of interest is pageSize working without specifying the page itself? That would be useful if for example I only want to display 3 things. So possibly uncoupling the parameters from each other

Reply:
I did it like this, so the user can't specify just a pageIndex. But after testing it, it even works with only the index set.

To you other question, yes that works that way.

Done